### PR TITLE
CS fixes

### PR DESCRIPTION
**Pull request by webimpress**
Updated CS rules + CS fixes (squizlabs/php_codesniffer: 2.6.2).
Also removed unused test (depends on non-existing test)

### CODE REVIEW

File: test/ContentValidationListenerTest.php [hunk 1]
```diff
@@ -1992,14 +1992,6 @@ public function testSaveUnknownDataWhenEmptyInputFilter()
     }
 
     /**
-     * @depends testFailsValidationOfPartialSetsForPatchRequestsForCollectionThatIncludeUnknownInputs
```
Comment (webimpress) [inline test/ContentValidationListenerTest.php]: This test does not exist, so below test was skipped.

Comment (weierophinney) [inline test/ContentValidationListenerTest.php]: I think this was supposed to refer to the test immediately above it; I'll verify, though.

Comment (weierophinney) [inline test/ContentValidationListenerTest.php]: argh! there's two similarly named test cases, and I evidently landed on the previous, which does not contain the "ForCollection" verbiage. So, yes, probably correct to remove it, though I'm curious if it should be present to test an edge case.

Comment (weierophinney) [inline test/ContentValidationListenerTest.php]: In checking through the tests, I think this behavior is already tested with:

- testReturnsApiProblemResponseForCollectionIfAnyFieldsAreInvalid
- testValidatePostedCollectionsAndAllowedOnlyFieldsFromFilterReturnsApiProblemWithUnrecognizedFields
- testReportsValidationFailureForPostedCollection
- testIndicatesInvalidPostedEntityWhenCollectionIsPossibleForService
- testFailsValidationOfPartialSetsForPatchRequestsThatIncludeUnknownInputs

as each demonstrates the return of an `ApiProblem` for invalid collections; the latter is also specifically targeting the unknown inputs situation.
